chart/kepler/values.yaml

```
@@ -78,3 +80,10 @@ serviceMonitor:
   interval: 30s
   scrapeTimeout: 5s
   labels: {}
+
+redfish:
+  enabled: false
+  name: redfish
+  labels: {}
+  annotations: {}
+  fileContent: "your_kubelet_node_name,redfish_username,redfish_password,https://redfish_ip_or_hostname"
```

It is better to be multiline as an example:

```suggestion
  fileContent: |-
    your_kubelet_node1_name,redfish_username,redfish_password,https://node1_redfish_ip_or_hostname
    your_kubelet_node2_name,redfish_username,redfish_password,https://node2_redfish_ip_or_hostname
```